main.go

```diff
@@ -22,6 +24,13 @@ func main() {
 	// parsing the flags has to be done after setting up all the flags
 	flag.Parse()
 
+	// If -fastq flag is not given, print the default usage message
+	if *rawSequenceFile == "dummy_sequence.fastq" {
+		panic(fmt.Sprintf("the flag -fastq <path/to/sequence.fastq> is required"))
+		flag.PrintDefaults()
+		os.Exit(1)
```

Comment:
These two lines will not happen because of the panic. Maybe move `flag.PrintDefaults()` above the panic and remove the `os.Exit(1)` call.

Reply:
👍